From 7aea4058a28529baf2a51931cfb9bf83d2d519e0 Mon Sep 17 00:00:00 2001
From: gnome-mpv <gnome-mpv@nowhere>
Date: Sun, 19 Aug 2018 01:06:54 -0700
Subject: [PATCH] player: Set default screenshot directory to XDG_PICTURES_DIR

Fixes #372.
---
 src/gmpv_player.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/gmpv_player.c b/src/gmpv_player.c
index b64c918..6c81562 100644
--- a/src/gmpv_player.c
+++ b/src/gmpv_player.c
@@ -381,6 +381,8 @@ static void apply_default_options(GmpvMpv *mpv)
 {
 	gchar *config_dir = get_config_dir_path();
 	gchar *watch_dir = get_watch_dir_path();
+	const gchar *screenshot_dir =	g_get_user_special_dir
+					(G_USER_DIRECTORY_PICTURES);
 
 	const struct
 	{
@@ -407,6 +409,7 @@ static void apply_default_options(GmpvMpv *mpv)
 			{"config", "no"},
 			{"config-dir", config_dir},
 			{"watch-later-directory", watch_dir},
+			{"screenshot-directory", screenshot_dir},
 			{"screenshot-template", "gnome-mpv-shot%n"},
 			{NULL, NULL} };
 
